oximeter/db/src/lib.rs

```diff
@@ -335,11 +335,13 @@ pub(crate) fn timeseries_key_for<'a>(
     target_fields: impl Iterator<Item = &'a Field>,
     metric_fields: impl Iterator<Item = &'a Field>,
 ) -> TimeseriesKey {
-    use std::collections::hash_map::DefaultHasher;
+    use highway::HighwayHasher;
     use std::hash::{Hash, Hasher};
-    let mut hasher = DefaultHasher::new();
+    let mut hasher = HighwayHasher::default();
     for field in target_fields.chain(metric_fields) {
```

Comment:
I've added this in 2e91139bb7699a1ca3e5e54306e8801a3aa23440 -- as expected, this does not change the hashes, since things are already sorted, but it does make it explicit.

I opted to alter the interface rather than re-sorting internally.